monitor.js

```diff
@@ -172,12 +172,12 @@ const Monitor = module.exports = {
 	countBattle(ip, name = '') {
 		let [count, duration] = this.battles.increment(ip, 30 * 60 * 1000);
 		if (duration < 5 * 60 * 1000 && count % 30 === 0) {
-			this.adminlog(`[ResourceMonitor] IP ${ip} has battled ${count} times in the last ${Chat.toDurationString(duration)}${name ? `: name` : ''})`);
+			this.adminlog(`[ResourceMonitor] IP ${ip} has battled ${count} times in the last ${Chat.toDurationString(duration)} ${name})`);
```

Comment:
@Zarel it wasn't displaying properly, for example here, instead of showing the name of the user battling too many times, it just showed "name" (someone forgot the `${}`). I've also simplified the code (name is `''` by default, it dosent need to be checked with an if.) and updated the statement that didn't use template strings.

Reply:
The lack of space is... not important, but nice to have.